sphinx_gallery/gen_rst.py

```diff
@@ -392,18 +405,18 @@ def scale_image(in_fname, out_fname, max_width, max_height):
                           generated images')
 
 
-def save_thumbnail(image_path, base_image_name, gallery_conf):
+def save_thumbnail(image_path, base_image_name, gallery_conf, thumb_num=1):
```

Comment:
The signature of the function is not great. It would be better if image_path was already the full path  of the image to be used for the thumbnail (and didn't need a .format).

Basically I would do:
```python
image_thumbnail_path = image_path.format(thumbnail_number)
```

in generate_file_rst.

image_path is not a great name, I would use something like image_path_template ... Bonus points if you fix that while you are touching this code.